File: 3581-the-two-sneaky-numbers-of-digitville/the-two-sneaky-numbers-of-digitville.py
class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        # nums_count = Counter(nums)
        # result = []
        # for num, freq in nums_count.items():
        #     if freq == 2:
        #         result.append(num)
        # return result

        nums_count = Counter(nums)
        return [num for num, freq in nums_count.items() if freq == 2]

File: 3581-the-two-sneaky-numbers-of-digitville/the-two-sneaky-numbers-of-digitville.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getSneakyNumbers = function(nums) {
    const ans = [];
    const freq = new Map();
    nums.forEach(item => {
        freq.set(item, (freq.get(item) || 0) + 1);
    });
    for (let [key, value] of freq){
        if (value === 2){
            ans.push(key)
        }
    }
    return ans
};